.ladle/components.tsx

```diff
@@ -47,19 +37,7 @@ const wrapProvider = (Component: React.ComponentType<any>, props?: any) => {
   );
 };
 
-const providers = [
-  wrapProvider(QueryClientProvider, { client: queryClient }),
-  wrapProvider(GrazProvider),
-  wrapProvider(WagmiProvider, { config }),
-  wrapProvider(LocaleProvider),
-  wrapProvider(RestrictionProvider),
-  wrapProvider(DydxProvider),
-  wrapProvider(AccountsProvider),
-  wrapProvider(SubaccountProvider),
-  wrapProvider(DialogAreaProvider),
```

Comment:
don't need dialogs or locale or whatever graz is?

Reply:
I kept locale.

We don't have any stories that include Dialog.
Graz is the Cosmos version of Wagmi. Removed it since we don't need any wallet providers